src/main/java/LeaveGroup/LeaveGroupPresenter.java

```diff
@@ -0,0 +1,13 @@
+package LeaveGroup;
+
+public class LeaveGroupPresenter implements LeaveGroupOutputBoundary {
```

Comment:
Again, what is the purpose of the presenter?

Reply:
review comment for presenter above ^